keg_apps/db/user-config-tpl-appveyor.py

```diff
@@ -0,0 +1,10 @@
+
+class TestProfile(object):
```

Comment:
Is this and the `cp` command in appveyour.yaml necessary? Can't we just use the default profile.